pkg/kubevirt/machine_test.go

```diff
@@ -453,7 +454,8 @@ var _ = Describe("util functions", func() {
 		machineContext.KubevirtMachine.Spec.VirtualMachineTemplate.Spec.DataVolumeTemplates = dataVolumeTemplates
 		machineContext.KubevirtMachine.Spec.VirtualMachineTemplate.Spec.Template.Spec.Volumes = volumes
 
-		newVM := newVirtualMachineFromKubevirtMachine(machineContext, "default")
+		newVM, err := newVirtualMachineFromKubevirtMachine(machineContext, "default")
+		Expect(err).To(BeNil())
```

Comment:
Nit: Please consider using the dedicated gomega error assertion.

```suggestion
		Expect(err).ToNnot(HaveOccurred())
```